### ISSUE TITLE
src directory missing which is causing warnings

### ISSUE THREAD

**Issue by sspilleman** [gevgeny/angular2-highcharts#10]
```
WARNING in ./~/angular2-highcharts/dist/index.js
Cannot find source file '../src/index.ts': Error: Cannot resolve 'file' or 'directory' ../src/index.ts in /Volumes/Data/Dropbox/Development/Node/ng2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/ChartComponent.js
Cannot find source file '../src/ChartComponent.ts': Error: Cannot resolve 'file' or 'directory' ../src/ChartComponent.ts in /Volumes/Data/Dropbox/Development/Node/ng2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/ChartSeriesComponent.js
Cannot find source file '../src/ChartSeriesComponent.ts': Error: Cannot resolve 'file' or 'directory' ../src/ChartSeriesComponent.ts in /Volumes/Data/Dropbox/Development/Node/ng2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/ChartPointComponent.js
Cannot find source file '../src/ChartPointComponent.ts': Error: Cannot resolve 'file' or 'directory' ../src/ChartPointComponent.ts in /Volumes/Data/Dropbox/Development/Node/ng2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/Highcharts.js
Cannot find source file '../src/Highcharts.ts': Error: Cannot resolve 'file' or 'directory' ../src/Highcharts.ts in /Volumes/Data/Dropbox/Development/Node/ng2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/HighchartsService.js
Cannot find source file '../src/HighchartsService.ts': Error: Cannot resolve 'file' or 'directory' ../src/HighchartsService.ts in /Volumes/Data/Dropbox/Development/Node/ng2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/initChart.js
Cannot find source file '../src/initChart.ts': Error: Cannot resolve 'file' or 'directory' ../src/initChart.ts in /Volumes/Data/Dropbox/Development/Node/ng2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/createBaseOpts.js
Cannot find source file '../src/createBaseOpts.ts': Error: Cannot resolve 'file' or 'directory' ../src/createBaseOpts.ts in /Volumes/Data/Dropbox/Development/Node/ng2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/deepAssign.js
Cannot find source file '../src/deepAssign.ts': Error: Cannot resolve 'file' or 'directory' ../src/deepAssign.ts in /Volumes/Data/Dropbox/Development/Node/ng2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/ChartEvent.js
Cannot find source file '../src/ChartEvent.ts': Error: Cannot resolve 'file' or 'directory' ../src/ChartEvent.ts in /Volumes/Data/Dropbox/Development/Node/ng2-seed/node_modules/angular2-highcharts/dist
```

**Comment by gevgeny** [comment]
Could you provide more details ?

**Comment by sspilleman** [comment]
1. git clone https://github.com/angular/angular2-seed.git
2. cd angular2-seed/
3. npm install
4. npm install angular2-highcharts --save
5. **modify files as below**
6. npm start

about.ts:

``` javascript
import {Component} from '@angular/core';
import {Http} from '@angular/http';
import { CHART_DIRECTIVES } from 'angular2-highcharts';

@Component({
  selector: 'about',
  templateUrl: 'app/components/about/about.html',
  styleUrls: ['app/components/about/about.css'],
  providers: [],
  directives: [CHART_DIRECTIVES],
  pipes: []
})
export class About {

  private options: Object;

  constructor(http: Http) {
    this.options = {
      title: { text: 'simple chart' },
      series: [{
        data: [29.9, 71.5, 106.4, 129.2],
      }]
    };
  }

  ngOnInit() {

  }
}
```

about.html

``` html
<h3>About Component</h3>
<p>This is the about component!</p>
<chart [options]="options"></chart>
```

warnings:

```
WARNING in ./~/angular2-highcharts/dist/index.js
Cannot find source file '../src/index.ts': Error: Cannot resolve 'file' or 'directory' ../src/index.ts in /private/tmp/angular2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/ChartComponent.js
Cannot find source file '../src/ChartComponent.ts': Error: Cannot resolve 'file' or 'directory' ../src/ChartComponent.ts in /private/tmp/angular2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/ChartSeriesComponent.js
Cannot find source file '../src/ChartSeriesComponent.ts': Error: Cannot resolve 'file' or 'directory' ../src/ChartSeriesComponent.ts in /private/tmp/angular2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/ChartPointComponent.js
Cannot find source file '../src/ChartPointComponent.ts': Error: Cannot resolve 'file' or 'directory' ../src/ChartPointComponent.ts in /private/tmp/angular2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/Highcharts.js
Cannot find source file '../src/Highcharts.ts': Error: Cannot resolve 'file' or 'directory' ../src/Highcharts.ts in /private/tmp/angular2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/HighchartsService.js
Cannot find source file '../src/HighchartsService.ts': Error: Cannot resolve 'file' or 'directory' ../src/HighchartsService.ts in /private/tmp/angular2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/initChart.js
Cannot find source file '../src/initChart.ts': Error: Cannot resolve 'file' or 'directory' ../src/initChart.ts in /private/tmp/angular2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/createBaseOpts.js
Cannot find source file '../src/createBaseOpts.ts': Error: Cannot resolve 'file' or 'directory' ../src/createBaseOpts.ts in /private/tmp/angular2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/deepAssign.js
Cannot find source file '../src/deepAssign.ts': Error: Cannot resolve 'file' or 'directory' ../src/deepAssign.ts in /private/tmp/angular2-seed/node_modules/angular2-highcharts/dist

WARNING in ./~/angular2-highcharts/dist/ChartEvent.js
Cannot find source file '../src/ChartEvent.ts': Error: Cannot resolve 'file' or 'directory' ../src/ChartEvent.ts in /private/tmp/angular2-seed/node_modules/angular2-highcharts/dist
```

**Comment by ckmilse** [comment]
have solved? I have an old project, webpack 1.2.1 ,still have the problem
